packages/teams-js/test/public/people.spec.ts

```diff
@@ -155,7 +155,9 @@ describe('people', () => {
   });
 
   describe('Testing people.isSupported function', () => {
+    let utils: Utils = new Utils();
```

Comment:
Similarly to my question in `teams.spec.ts`: what about your change made this required / helpful?

Reply:
This is the same as the above